CoEpi/AppDelegate.swift

```diff
@@ -23,6 +24,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let bgManager: BackgroundTasksManager = try! container.resolve()
         bgManager.onApplicationFinishLaunching()
 
+        let notificationShower: NotificationShower = try! container.resolve()
+        notificationShower.scheduleReminderNotificationsForNext(days: 14)
```

Comment:
Number of days for which tp schedule reminder notifications (in advance). 
iOS limits number of pending local notifications to 64.